Shadan/style.css

```diff
@@ -0,0 +1,159 @@
+*
+{
+    margin:0;
+    padding:0;
+}
+html{
+    background-color: #f7f7f7;
+}
+header{
+    background-color: #e6e6e6;
+
+}
+
+header .topcenter{
+    position:relative;
+}
+
+nav .logo{
+
+    width:50px;
+    height:50px;
+   position: absolute;
+   left:25%;
+}
+header nav .navbar{
+    margin-top:20px;
+    text-align: center;
+     position: absolute;
+     right:25%;
+     
+}
+nav .navbar a{
+    text-decoration: none;
+    border: 1px solid #1DA1F2;
+    border-radius: 12px;
+    padding: 6px;
+    margin-top:10px;
+     margin:5px;
+     padding-right:20px;
+     padding-left:20px;
+    text-transform:uppercase;
+}
+a.white{
+    color:#1DA1F2;
+
+}
+a.blue{
+    background-color:#1DA1F2;
+    color:white;
+}
+.clearfix::after {
+    content: "";
+    clear: both;
+    display: table;
+  }
+.header-hr{
+    margin-top:60px;
+    border:none;
+    border-bottom:1px solid #8c8c8c;    ;
+    box-shadow:1px 1px 2px  #8c8c8c;
+}
+main .all-box{
+    box-shadow:1px 1px 2px #666666;
+    width:60%;
+    height:100%;
+    padding:0;
+    margin:0;
+    position:relative;
+    margin-left:20%;
+    background-color: white;
+    z-index: -1;
+}
+main .coverimg{
+    width:100%;
+    height:500px;
+    position:relative;
+    top:-100px;
```

Comment:
I don't think that is good practice to use `top` property or even `left`, `right` and `bottom` with  `position:relative`
the best practice here is to gave the parent `position: relative` and then give this one `position:absolute` and play with the position